api/src/services/auth.service.ts

```diff
@@ -356,7 +356,10 @@ export class AuthService {
   async confirmUser(dto: Confirm, res?: Response): Promise<SuccessDTO> {
     const token = verify(dto.token, process.env.APP_SECRET) as IdAndEmail;
 
-    let user = await this.userService.findUserOrError({ userId: token.id });
+    let user = await this.userService.findUserOrError(
+      { userId: token.id },
+      UserViews.full,
+    );
```

Comment:
I think this UserViews.full is passed to all instances of findUserOrError. Integration tests were failing for this